Documentation/Usage.md

```diff
@@ -482,7 +487,7 @@ AF.request("https://httpbin.org/get").response { response in
 
 #### Response Data Handler
 
-The `responseData` handler uses a `DataResponseSerializer`  to extract and validate the `Data` returned by the server. If no errors occur and `Data` is returned, the response `Result` will be a `.success` and the `value` will be the `Data` returned from the server.
+The `responseData` handler uses a `DataResponseSerializer`to extract and validate the `Data` returned by the server. If no errors occur and `Data` is returned, the response `Result` will be a `.success` and the `value` will be the `Data` returned from the server.
```

Comment:
Now there's no space between `DataResponseSerializer` and `to`.

Reply:
Yep, that was my bad, fixed in 68676b8d.